src/Headers/Hydrodynamics.h

```diff
@@ -50,7 +51,7 @@ class EOS;
 
 class SimulationBase;
 
-static const FLOAT ghost_range = 2.5;
+static const FLOAT ghost_range = 3.0; //2.5;
```

Comment:
Yes, I'd prefer it smaller for obvious reasons.  But I also don't like the Travis tests being so sensitive to failing like these ones.